### ISSUE TITLE
Cloudinary image gallery : labels and reordering

### ISSUE THREAD

**Issue by glimow** [keystonejs/keystone#4808]
I have used keystone in production and my customers are pretty happy with it. However, a significant part of them expected the ability to reorder images in the cloudinary gallery and to add label on them. I patched this (in a pretty dirty way) on my keystone, but I think that would be an interesting supported feature as most CMSs offer them.

**Comment by Antekirta** [comment]
@glimow, could you tell us about your "dirty approach"? I'm very intrigued.

**Comment by abhijithvijayan** [comment]
@glimow I did similar approach on https://stanlywilson.me/work

It doesn't need any hard coding. Assigned a tag property to each image and a priority number to each image and sorted it out based on a category.


It would be quite useful if keystone had something like that in it build it.

**Comment by glimow** [comment]
@abhijithvijayan I'm not sure I fully understand, You didn't do any any modification to keystone's code to achieve that ? That means you have not used the `CloudinaryImages` field but rather a List of `CloudinaryImage` along with other fields, am I wrong ?

**Comment by abhijithvijayan** [comment]
@glimow oh sorry, I didn't notice the CloudinaryImage(s) field, rather I used like you said a list of CloudinaryImage fields.

but the gallery view seems way better than my approach. Do you have a demo or something so as to check it out.

Thanks.

**Comment by glimow** [comment]
Yeah the gallery is pretty cool, especially because I had to integrate a per-page gallery in my project. Unfortunately my code is a dirty hack and It's not very suitable for a demo or smth. But I'm up helping integrate that into keystone. The true problem is that keystone doesn't support nested lists, but I think this is not in the roadmap because there are other priorities.
